src/apps/allbridge/api.ts

```diff
@@ -98,7 +98,11 @@ export async function getAllbridgeTokenInfo({
   ) as AllbridgeApiResponse
   if (!allbridgeTokensInfoResponse) {
     allbridgeTokensInfoResponse = await got
-      .get('https://core.api.allbridgecoreapi.net/token-info')
+      .get('https://core.api.allbridgecoreapi.net/token-info', {
+        headers: {
+          'x-Sdk-Agent': 'AllbridgeCoreSDK/0.0.0-development', // as in https://github.com/allbridge-io/allbridge-core-js-sdk/blob/5deb0623d6fffcbc7a85dba22b98942c47d4af6c/src/client/core-api/api-client.ts#L48
```

Comment:
Looks like the shipped libs have a real value: `3.21.0` for the latest version
https://www.npmjs.com/package/@allbridge/bridge-core-sdk?activeTab=code

<img width="611" alt="Screenshot 2025-02-25 at 14 58 36" src="https://github.com/user-attachments/assets/e80dcb39-dc33-47b0-9bb7-ea6ccaf40bd9" />